i18n/en.json

```diff
@@ -19,6 +19,7 @@
 	"login": "Log in",
 	"logout": "Log out",
 	"logout-msg": "Currently logged in as $1",
+    "markdown": "Markdown",
```

Comment:
should be a tab, not spaces (the translatewiki.net bot might fix this automatically)